.github/workflows/continuous_integration.yml

```diff
@@ -4,9 +4,6 @@ on:
   push:
     branches:
       - master
-  pull_request:
```

Comment:
It would be nice to leave this and exclude deploy steps.
It will check for valid markdown etc.

cmp https://github.com/QGEP/docs/blob/master/.github/workflows/release.yml#L19

Reply:
oh, ignore, that's the cron

Reply:
actually, that might be a nice enhancement of  line 28 (check the event for push)